src/sentry/api/serializers/models/release_file.py

```diff
@@ -29,8 +30,8 @@ def decode_release_file_id(id: str):
     except ValueError:
         decoded = urlsafe_b64decode(id).decode()
         dist, url = decoded.split("_", 1)
-        dist = dist or None
-        return dist, url
+        ret_dist: Optional[str] = dist or None
+        return ret_dist, url
```

Comment:
probably don't need the annotation now that it's renamed -- can probably also be `return dist or None, url` without needing a separate variable